test/breez_translations_test.dart

```diff
@@ -906,6 +906,102 @@ void main() {
         expect(text.contains(minSats), true);
         expect(text.contains(maxSats), true);
       });
+
+      test("satscard_error_invalid_body for ${locale.locale}", () {
```

Comment:
Thanks for adding the tests :pray:

Reply:
If there's anything else missing let me know, I tried to follow the standard practices of your app but it's a large codebase so I likely missed some things ❤️